dev/wlp-gradle/subprojects/tasks.gradle

```diff
@@ -208,7 +208,7 @@ if ( parseBoolean( bnd('jakartaeeMe', 'false'))) {
           '-tv', transformProject.getFile('rules/jakarta-versions.properties').path,
           '-tb', transformProject.getFile('rules/jakarta-bundles.properties').path,
           '-td', transformProject.getFile('rules/jakarta-direct.properties').path,
-          '-tf', transformProject.getFile('rules/jakarta-xml-master.properties').path ]
```

Comment:
Although this reference has changed. It looks like the jakarta-xml-master.properties file itself was not removed?

Reply:
This was on purpose to not have to do a combined build.  So instead doing a 3 PR change.  This one for OL, then a PR for WebSphere Liberty to remove the reference to jakarta-xml-master and then a new one for OL to delete jakarta-xml-master.

Reply:
This was unhelpful as we'd made changes to this file in #18943 which stopped working when this PR was was merged.

Because the original file is still there we didn't get a merge conflict and it took some time to work out what had changed.

Reply:
Got it, thanks, it was an annoying soft merge conflict but I guess that can't really be helped in this case.